5_data_munge/src/qaqc_daily_data.R

```diff
@@ -0,0 +1,62 @@
+############ Function to read the daily temperature and site ind data #######
```

Comment:
This isn't a big deal -- but if you're going through the trouble of documenting, you should say what the function is doing. The purpose of the function is to QAQC the daily temperature data using XYZ methods.